stix_shifter_modules/azure_log_analytics/stix_transmission/connector.py

```diff
@@ -1,5 +1,6 @@
 import json
 from stix_shifter_utils.modules.base.stix_transmission.base_sync_connector import BaseSyncConnector
```

Comment:
Is this still used or can it be removed?

Reply:
ohh forgot to remove it.